docs/README.md

```diff
@@ -1,20 +0,0 @@
-# Covasim docs
```

Comment:
Yep, I agree. Just doing a lot of experimentation on this PR to figure out why the build was breaking. I suspected it might be because md files were ignored before myst was added, but now they are considered a source file that's missing from the TOC. With myst implemented, we can add it to the exclude list in conf.py.